proxy/src/serverless/rest.rs

```diff
@@ -0,0 +1,1223 @@
+use super::backend::HttpConnError;
```

Comment:
For proxy we use the following command to do one-off formatting of imports after larger changes:

```shell
cargo +nightly fmt -p proxy -- -l --config imports_granularity=Module,group_imports=StdExternalCrate,reorder_imports=true
```

Please run.